### DISCUSSION TITLE
Want to use neovis with out neo4j

### DISCUSSION THREAD

**Discussion by subashw** [neo4j-contrib/neovis.js#282] [Q&A]
Hi

I want to use neovis without neo4j driver dependencies

Is that possible?

Can someone point me to example code?

Basically instead of passing in a cypher query to neovis I want to send/stream a dataset.

Is that possible

Regards

Subash Warrier

**Comment by thebestnom** [comment]
Not currently, it's an open issue (I can think of a way to hack that in without changing the code, but I don't think like a good idea to share it 😅)

**Reply by subashw** [reply]
If we I send you the method to do so in a private email I am happy to send you my email.  I need it badly.

**Comment by subashw** [comment]
never mind I skipped neovis.js altogether and I used vis.js.  I am able to stream results.

**Reply by subashw** [reply]
This also has one more benefit I don't have to use bolt.  I can use bolt from from backend code using localhost:7687.  I can also keep all connections to the eternal world ssl/tls reverse proxy to stream results over https.
